fastlane/Fastfile

```diff
@@ -219,3 +307,32 @@ def replace_in(previous_text, new_text, path, allow_empty=false)
   backup_extension = '.bck'
   sh("sed", '-i', backup_extension, sed_regex, path)
 end
+
+def parse_previous_android_native_version
+  parse_version_in_file(/(?<=ext\.purchases_version = ["'])(.*)(?=["'])/, "android/build.gradle")
```

Comment:
One thing I was thinking about is that we can maybe use the `'fastlane-plugin-versioning_android'` plugin to get the current version, same way we do in the `purchases-android` repo... That said, I think it's better to keep it as you have it, since it's the only usage, and that wouldn't work in the case of iOS.

Reply:
that gives you the current version of the project, but not the current version of the dependency, which is what we want in this case. So I don't think we can use that plugin